crates/fluvio/src/consumer/mod.rs

```diff
@@ -348,12 +366,20 @@ where
         if stream_fetch_version < CHAIN_SMARTMODULE_API {
             warn!("SPU does not support SmartModule chaining. SmartModules will not be applied to the stream");
         }
+        if stream_fetch_version < OFFSET_MANAGEMENT_API {
```

Comment:
This warning should be only applied if consumer id is not null

Reply:
Yes, indeed. Fixed.